ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Embed/Nary.hs

```diff
@@ -205,10 +206,11 @@ injectInitialExtLedgerState cfg extLedgerState0 =
 
     targetEraLedgerState :: LedgerState (HardForkBlock (x ': xs))
     targetEraLedgerState =
-        HardForkLedgerState $
+        -- TODO discarding ledger events, probably fine?
```

Comment:
I would also guess it's fine, but it's slightly tricky since this whole function is an (important) optimization (for the benchmarking team)

Reply:
I left a justification.